docs/user/resources/01-serverless.md

```diff
@@ -0,0 +1,104 @@
+
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **conditions**  | \[\]object | Conditions associated with CustomStatus. |
+| **conditions.&#x200b;lastTransitionTime** (required) | string | Specifies the last time the condition transitioned from one status to another. This should be when the underlying condition changes.  If that is not known, then using the time when the API field changed is acceptable. |
+| **conditions.&#x200b;message** (required) | string | a human-readable message indicating details about the transition. This may be an empty string. |
+| **conditions.&#x200b;observedGeneration**  | integer | Represents the **.metadata.generation** that the condition was set based upon. For instance, if **.metadata.generation** is currently `12`, but the **.status.conditions[x].observedGeneration** is `9`, the condition is out of date with respect to the current state of the instance. |
+| **conditions.&#x200b;reason** (required) | string | Contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field and whether the values are considered a guaranteed API. The value should be a camelCase string. This field may not be empty. |
+| **conditions.&#x200b;status** (required) | string | status of the condition, one of `True`, `False`, `Unknown`. |
+| **conditions.&#x200b;type** (required) | string | type of condition in camelCase or in `foo.example.com/CamelCase`. --- Many **.condition.type** values are consistent across resources like `Available`, but because arbitrary conditions can be useful (see **.node.status.conditions**), the ability to deconflict is important. The regex it matches is `(dns1123SubdomainFmt/)?(qualifiedNameFmt)` |
+| **dockerRegistry**  | string | Used registry configuration. Contains registry URL or "internal" |
+| **eventingEndpoint**  | string | Used Eventing endpoint. |
+| **served** (required) | string | Served signifies that current Serverless is managed. Value can be one of `True`, or `False`. |
+| **state**  | string | Signifies the current state of Serverless. Value can be one of `Ready`, `Processing`, `Error`, or `Deleting`. |
+| **tracingEndpoint**  | string | Used Tracing endpoint. |
+
+<!-- TABLE-END -->
+
+### Status reasons
+
+Processing of a Serverless CR can succeed, continue, or fail for one of these reasons:
+
+
+# Serverless CR conditions
+
+This section describes the possible states of the Serverless CR. Three condition types, `Installed`, `Configured` and `Deleted`, are used.
+
+| No | CR State   | Condition type | Condition status | Condition reason      | Remark                                        |
+|----|------------|----------------|------------------|-----------------------|-----------------------------------------------|
+| 1  | Ready      | Configured     | true             | Configured            | Serverless configuration verified             |
```

Comment:
Configured true condition informs us that the installation can be processed, but the Serverless is not Ready. It's still unknown because in the model use-case (first installation) it's still not installed (Processing)
```suggestion
| 1  | Processing | Configured     | true             | Configured            | Serverless configuration verified             |
```

Reply:
done